clients/drcachesim/tools/record_view.h

```diff
@@ -0,0 +1,96 @@
+/* **********************************************************
```

Comment:
Maybe these files should be renamed to internal_record_review* as well so someone looking at the file structure doesn't see "record_view.cpp" and assume "that's what I should be using".

Reply:
Done.